MekHQ/src/mekhq/gui/StratconTab.java

```diff
@@ -291,6 +295,8 @@ private String buildShortStrategicObjectiveText(StratconCampaignState campaignSt
     private String buildStrategicObjectiveText(StratconCampaignState campaignState) {
         StringBuilder sb = new StringBuilder();
 
+        boolean contractIsActive = campaignState.getContract().isActiveOn(getCampaignGui().getCampaign().getLocalDate());
```

Comment:
Variable is useless given it is used a grand total of once, and isn't required at all times.